api/pkg/s3/bucketget.go

```diff
@@ -145,30 +155,18 @@ func CreateListObjectsResponse(bucketName string, request *s3.ListObjectsRequest
 		response.Contents = append(response.Contents, obj)
 	}
 
-	var prefixes []datatype.CommonPrefix
```

Comment:
Hi @rhsakarpos is it compatible with aws s3 if remove those codes?

Reply:
hi @sfzeng i need to check the AWS spec and align this(with support for all cloud backends) as needed, since now, we are NOT setting versioning on the cloud backend. I plan to do this after RC1, does that work ? I'll raise an issue to track it for RC2 ?